docs/intro/dev-portal/link-github-id.md

```diff
@@ -19,12 +19,12 @@ To link a new GitHub ID, you’ll first have to unlink the current one. Although
 1. Sign in to the Gruntwork Developer Portal and navigate to your [Profile Settings](https://app.gruntwork.io/settings/profile).
 2. Click **Unlink** in the description under the **GitHub Account** section.
 3. Click **Yes, Unlink My Account** in the confirmation dialog that appears.
-4. Proceed with the [steps above](#linking-your-github-account) to link a new GitHub account.
+4. Proceed with the [steps above](#linking-your-github-account) to link a new GitHub account. **For best results, we recommend performing these steps in an incognito browser window. This will ensure that you can use the exact GitHub account that you want to link.**
```

Comment:
Thanks for putting this together, Ore! I recommend using italics instead of bold here.
```suggestion
4. Proceed with the [steps above](#linking-your-github-account) to link a new GitHub account *using a private/incognito browser window*. (This guarantees you'll have an opportunity to specify the new account you wish to link.)
```